src/content/blog/community-day-2023.mdx

```diff
@@ -0,0 +1,188 @@
+  img={import('./_images/community-day-2023/automation.webp')}
+/>
+
+<CommunityProject 
+  creator="Nate Moore"
+  github="https://github.com/natemoo-re"
+  name="Houston Theme"
+  link="https://github.com/withastro/houston-vscode"
+  linkTextOverride="github.com/withastro/houston-vscode"
+  description="An Astro-branded Visual Studio Code theme featuring cool blues, minty greens, and soft purples and Astro’s mascot Houston."
+  fact="Astro’s brand colors were begging to be turned into a syntax highlighting theme. With some adjustments for contrast and legibility, I am quite happy that this little experiment turned out so well. This theme has been downloaded by 12.5K people on the Visual Studio Code marketplace* and is the default theme for Astro’s Starlight documentation framework. (*Editors note: and 169 on the OpenVSX registry! 😉)"
+  img={import('./_images/community-day-2023/nate.webp')}
+/>
+
+<CommunityProject 
+  creator="happydev"
+  github="https://github.com/moustaphadev"
+  name="Live Astro Compiler"
+  link="https://live-astro-compiler.vercel.app/"
+  linkTextOverride="live-astro-compiler.vercel.app"
+  description="A playground for the Astro compiler that generates insightful previews of the compiled output of Astro components."
+  fact="Sparked by the struggle of quickly spotting compiler bugs, I built this to save time on issue troubleshooting. Instead of diving straight into writing tests, now we can just toss some code into the playground, check out the output, and easily spot and show others what’s going wrong — making the whole debugging dance way smoother!"
+  img={import('./_images/community-day-2023/happydev.webp')}
+/>
+
+<CommunityProject 
+  creator="Chris"
+  github="https://github.com/delucis/"
+  name="Astro Badges"
+  link="https://astro.badg.es/"
+  linkTextOverride="astro.badg.es"
+  description="Astro Badges celebrates Astro contributors with embeddable images featuring achievements unlocked by GitHub activity in the withastro organisation. From Little Green Bug to Constellation Crafter, every one of Astro’s almost 3,500 contributors is here!"
+  fact="Astro Badges started as a weekend project after I noticed the badg.es domain was available. It’s grown a lot since then, including a full redesign, collaborating with Astro’s designer, @doodlemarks. Probably the most challenging thing about maintaining it is coming up with fun achievement names, although @sarah11918 is always game to workshop them! My favourite recent addition is probably the SSRsly achievement for people working on Astro’s serverless adapters. Shout out to @Yan-Thomas, @ElianCodes, @natemoo-re, @mayank99, @torn4dom4n, @jacobthesheep, and @TheOtterlord for contributing along the way 🙌"
```

Comment:
```suggestion
  description="Astro Badges celebrates Astro contributors with embeddable images featuring achievements unlocked by GitHub activity in the withastro organization. From Little Green Bug to Constellation Crafter, every one of Astro’s almost 3,500 contributors is here!"
  fact="Astro Badges started as a weekend project after I noticed the badg.es domain was available. It’s grown a lot since then, including a full redesign in collaboration with Astro’s designer, @doodlemarks. Probably the most challenging thing about maintaining it is coming up with fun achievement names, although @sarah11918 is always game to workshop them! My favorite recent addition is probably the SSRsly achievement for people working on Astro’s serverless adapters. Shout out to @Yan-Thomas, @ElianCodes, @natemoo-re, @mayank99, @torn4dom4n, @jacobthesheep, and @TheOtterlord for contributing along the way 🙌"
```